modules/preview-react/radio/stories/examples/Inverse.tsx

```diff
@@ -4,13 +4,20 @@ import {RadioGroup} from '@workday/canvas-kit-preview-react/radio';
 import {FormField} from '@workday/canvas-kit-preview-react/form-field';
 import {styled, StyledType} from '@workday/canvas-kit-react/common';
 import {colors} from '@workday/canvas-kit-react/tokens';
+import {createStyles} from '@workday/canvas-kit-styling';
+import {base, system} from '@workday/canvas-tokens-web';
 
 const StyledFormField = styled(FormField)<StyledType>({
   legend: {
     color: colors.frenchVanilla100,
   },
 });
 
+const parentContainerStyles = createStyles({
+  backgroundColor: base.blueberry400,
```

Comment:
should this be system token? I guess we don't need to support of using base tokens if there is system alternative existing

Reply:
Yup!